cfbs/commands.py

```diff
@@ -20,6 +20,7 @@
     user_error,
     strip_right,
     pad_right,
+    ProgrammerError,
```

Comment:
Where is the new exception being added?

Reply:
It was added in the first commit of this PR, but I can rewrite history to make it more clear :)